_data/retail.yml

```
@@ -199,16 +232,22 @@ websites:
     img: homedepot.png
     twitter: HomeDepot
     facebook: homedepot
+    regions:
+      - us
```

```suggestion
    regions:
      - us
      - ca
      - mx
```
Home Depot also serves [Canada](https://www.homedepot.ca) and [Mexico](https://www.homedepot.com.mx).